src/View3d.ts

```diff
@@ -223,6 +227,9 @@ export class View3d {
 
       if (unit) {
         this.image.volume.setUnitSymbol(unit);
+      } else if (!isOrthographicCamera(this.canvas3d.camera)) {
+        // `else if` because `setUnitSymbol` would have done this already
```

Comment:
I'm not sure I understand this comment

Reply:
Hmm... actually, the `else if` statement it's referencing shouldn't be there in the first place. I thought the function called on 229 would call `updatePerspectiveScaleBar`, and therefore this function should only check to call it itself if no unit was provided (this is what the comment was supposed to mean). Not only does the function on 229 not call `updatePerspectiveScaleBar`, but `updatePerspectiveScaleBar` should be called regardless of whether the current camera is orthographic or not.